packages/gsul/src/dynamoDBclient.ts

```diff
@@ -39,7 +39,7 @@ export async function getItemsUpdatesForPrescription(
   return items.map((singleUpdate) => ({
     itemId: String(singleUpdate.LineItemID),
     latestStatus: String(singleUpdate.Status),
-    isTerminalState: String(singleUpdate.TerminalStatus),
+    isTerminalState: Boolean(singleUpdate.TerminalStatus),
```

Comment:
the data in dynamo will still be a string, we need to add `=== "completed"` here

Reply:
A check to determine whether the item has reached the terminal state `completed` has been implemented.